src/inspector/node_protocol.pdl

```diff
@@ -92,3 +92,12 @@ experimental domain NodeWorker
       # Identifier of a session which sends a message.
       SessionID sessionId
       string message
+
+experimental domain NodeRuntime
+  command enable
```

Comment:
Please provide more details on what this does.

Reply:
done!